framework/core/js/src/forum/components/SecurityPage.tsx

```diff
@@ -144,8 +162,8 @@ export default class SecurityPage<CustomAttrs extends IUserPageAttrs = IUserPage
         <Button
           className="Button"
           onclick={this.terminateAllOtherSessions.bind(this)}
-          loading={this.loading}
-          disabled={!this.tokens?.find((token) => token.isSessionToken() && !token.isCurrent())}
+          loading={this.loading === 'terminate_sessions'}
+          disabled={this.loading === 'global_logout' || !this.tokens?.find((token) => token.isSessionToken() && !token.isCurrent())}
```

Comment:
Not code from this PR, but this seems to be what was desired.

```suggestion
          disabled={this.loading === 'global_logout' || !this.tokens?.some((token) => token.isSessionToken() && !token.isCurrent())}
```